src/mmw/js/src/draw/views.js

```diff
@@ -568,6 +573,11 @@ var ResetDrawView = Marionette.ItemView.extend({
     },
 
     resetDrawingState: function() {
+        App.getLeafletMap().eachLayer((l) => {
+            if (l.id && l.id === 'rwd-origin-point') {
+                App.getLeafletMap().removeLayer(l);
+            }
+        });
```

Comment:
Oh, I must have skipped placing it in state in the first place. I'll see about adding it to the `rwdTaskModel` state, which would obviate needing to do the find by id stuff.